test/requests/shelters/shelterRequest-test.ts

```diff
@@ -69,6 +77,56 @@ describe('Post Shelter', () => {
 
 
       // expect structure of the response
+      expect(res.body.data.id).to.exist;
+      expect(res.body.data.id).to.be.a('number');
+      expect(res.body.data.type).to.equal('shelter');
+      expect(res.body.data.attributes.name).to.equal('Golden Sun');
+      expect(res.body.data.attributes.streetAddress).to.equal('1234 Black St');
+      expect(res.body.data.attributes.state).to.equal('NY');
+      expect(res.body.data.attributes.zip).to.equal(78123);
+      expect(res.body.data.attributes.phoneNumber).to.equal('2134568765');
+      expect(res.body.data.attributes.websiteUrl).to.equal(null);
+      expect(res.body.data.attributes.verified).to.equal(false);
+      done();
+    })
+  })
+
+  it('create a shelter given a name and address and phone number AND a websiteurl', (done) => {
+    const shelterParams: ShelterPost = {
+      name: 'Golden Sun',
+      streetAddress: '1234 Black St',
+      state: 'NY',
+      zip: 78123,
+      phoneNumber: '2134568765',
+      websiteUrl: "www.fake.com"
+    }
+
```

Comment:
This file has had some significant changes, and the version on main is correct. This file is not related to this PR and is out of date.